src/Microsoft.AspNet.Identity/IdentityLogger.cs

```diff
@@ -0,0 +1,46 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Runtime.CompilerServices;
+using Microsoft.Framework.Logging;
+
+namespace Microsoft.AspNet.Identity
+{
+    public class IdentityLogger
```

Comment:
I prefer that we keep things simple by making it a static class with extension utility methods (and we move it to a nested namespace). It doesn't seem that compelling to replace it and BTW, this PR does not make it because we are creating new instances with `new` as opposed to making it a service.